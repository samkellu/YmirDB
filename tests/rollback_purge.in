DROP 1
ROLLBACK 1
SET a 1 2
SET b 3 4
LIST ENTRIES
SNAPSHOT
SET c 5 6
LIST ENTRIES
SNAPSHOT
PURGE b
LIST ENTRIES
ROLLBACK 1
LIST SNAPSHOTS
GET b
SET b 123 33 a
SNAPSHOT
BYE
