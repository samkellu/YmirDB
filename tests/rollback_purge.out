> no such snapshot

> no such snapshot

> ok

> ok

> b [3 4]
a [1 2]

> saved as snapshot 1

> ok

> c [5 6]
b [3 4]
a [1 2]

> saved as snapshot 2

> ok

> ok

> 1

> bye
